hail/Makefile

```diff
@@ -207,6 +207,24 @@ deploy: tag
 install-deps:
 	$(PIP) install -U -r python/requirements.txt -r python/dev-requirements.txt
 
+BENCHMARK_DOCKER_TAG := benchmark_$(shell whoami)
+PROJECT = $(shell gcloud config get-value project)
```

Comment:
this is lazily expanded and used in multiple places, but I don't want to make it `:=` because that slows down all `make` targets by about 500ms.